File: 800/CF99A_Help_Far_Away_Kingdom.py
s = input()

integer_part, fraction_part = s.split('.')

# If integer part ends with 9
if integer_part[-1] == '9':
    print("GOTO Vasilisa.")

# Fractional part < 0.5
elif fraction_part[0] < '5':
    print(integer_part)

# Fractional part >= 0.5
else:
    last = int(integer_part[-1])
    last += 1

    integer_part = integer_part[:-1] + str(last)

    print(integer_part)

File: 800/CF99A_Help_Far_Away_Kingdom.java

import java.util.*;

public class CF99A_Help_Far_Away_Kingdom {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String s = sc.nextLine();

        String[] parts = s.split("\\.");
        String integerPart = parts[0];
        String fractionPart = parts[1];

        // If integer part ends with 9
        if (integerPart.charAt(integerPart.length() - 1) == '9') {
            System.out.println("GOTO Vasilisa.");
            return;
        }

        // Fractional part is less than 0.5
        if (fractionPart.charAt(0) < '5') {
            System.out.println(integerPart);
        } 
        else {
            // Round up
            char last = integerPart.charAt(integerPart.length() - 1);
            last++;

            integerPart = integerPart.substring(0, integerPart.length() - 1) + last;

            System.out.println(integerPart);
        }
    }
}